stripe/src/test/java/com/stripe/android/StripeTest.java

```diff
@@ -19,6 +20,7 @@
  * Test class for {@link Stripe}.
  */
 @RunWith(RobolectricTestRunner.class)
+@Config(sdk = 23)
```

Comment:
Should probably make the config sdk 25 since that is the target sdk of the project.